samples/net/mdns_responder/src/service.c

```diff
@@ -66,7 +66,7 @@ void service(void)
 #else
 	/* The simple use case: fixed port */
 	DNS_SD_REGISTER_TCP_SERVICE(zephyr, CONFIG_NET_HOSTNAME,
-				    "_zephyr", "local", DNS_SD_EMPTY_TXT, DEFAULT_PORT);
+					"_zephyr", "local", DNS_SD_EMPTY_TXT, DEFAULT_PORT);
```

Comment:
This looks like unrelated change and the line is properly indented already, please remove it.

Reply:
I had to convert spaces to tabs to fix indentation issues from my changes, it affected the entire file. I will revert this change.

Reply:
Fixed